tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -25,8 +25,9 @@
     "dcv_port, access_from, shared_dir", [(8443, "0.0.0.0/0", "/shared"), (5678, "192.168.1.1/32", "/myshared")]
 )
 @pytest.mark.regions(["eu-west-1", "cn-northwest-1"])  # DCV license bucket not present in us-gov
-@pytest.mark.oss(["centos7"])
-@pytest.mark.schedulers(["sge", "awsbatch"])
+@pytest.mark.oss(get_supported_dcv_os())
+@pytest.mark.instances(["c4.xlarge", "g3.8xlarge"])
```

Comment:
Probably we should add the g4 in place of the g3 since it is the one that requires a specific driver to work (the same is valid for slurm) what do you think?

Reply:
Some of the slurm_gpu tests are specific to `g3.8xlarge`(32 cpus/2 gpus), there is no instance in the g4 family with the same number of cpu/gpu. To change to g4 I will have to rewrite the logic we are testing in slurm_gpu tests. I would suggest to change to g4 later in a separate PR.